payment-platform-adapter/src/helpers/logger.ts

```diff
@@ -1,10 +1,10 @@
 /* eslint-disable @typescript-eslint/no-explicit-any, no-console */
 import { configuration } from '../config';
 
-const isDebugging = true; // configuration.dev === 'dev';
 
 export abstract class LoggerService {
   private static source = configuration.functionName;
+  public static isDebugging = configuration.dev === 'dev';
```

Comment:
I think this would be better. While Jest running the Env variable is 'test'.
```suggestion
  public static isDebugging = configuration.dev === 'dev' || 'test';
```